### PR TITLE
Zapier doc updates

### PR DESCRIPTION
**Pull request by esethna**
Minor updates including:
1. work around to "Is Trusted" bug
2. Deauthorizing Zapier

### CODE REVIEW

File: source/integrations/zapier.rst [hunk 1]
```diff
@@ -23,14 +23,14 @@ Enable OAuth 2.0
...
-  c. If you’d like to allow users on your system who are not Team or System Admins to create Zapier integrations, then set `Restrict creating integrations to Team and System Admins <https://docs.mattermost.com/administration/config-settings.html#restrict-creating-integrations-to-team-and-system-admins>`_ to **False**.
+  b. If you’d like to allow Zapier integrations to post with customizable usernames and profile pictures, then set `Enable integrations to override usernames <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-usernames>`_ and `Enable integrations to override profile picture icons <https://docs.mattermost.com/administration/config-settings.html#enable-integrations-to-override-profile-picture-iconss>`_ to **True**.
+  c. If you’d like to allow users on your system who are not Team or System Admins to create Zapier integrations, then set `Restrict managing integrations to Admins <https://docs.mattermost.com/administration/config-settings.html#restrict-managing-integrations-to-admins>`_ to **False**.
```
Comment (lfbrock) [inline source/integrations/zapier.rst]: Maybe change "create Zapier integrations" to "connect a Zapier account"?

File: source/integrations/zapier.rst [hunk 2]
```diff
@@ -145,6 +145,9 @@ Cannot connect a Mattermost account
 
   .. image:: ../images/zapier-error3.png
 
+Deauthorize the Zapier App
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+1. If you'd deauthorize Zapier so it can no longer post through your connected account, go to **Account Settings** > **Security** > **OAuth 2.0 Applications**, then click **Deauthorize** on the Zapier app.
```
Comment (lfbrock) [inline source/integrations/zapier.rst]: Should "you'd" be "you'd like to"

**Comment by esethna** [comment]
Tweaked, thanks!

**Comment by esethna** [comment]
Pushing this one in as the updated instructions are needed for Zapier to test our app. Leaving the 2nd review label on.

**Pull request merged**